Refresh.GameServer/Services/CommandService.cs

```diff
@@ -0,0 +1,103 @@
+using Bunkum.HttpServer;
+using Bunkum.HttpServer.Services;
+using JetBrains.Annotations;
+using MongoDB.Bson;
+using NotEnoughLogs;
+using Refresh.GameServer.Database;
+using Refresh.GameServer.Types.Commands;
+using Refresh.GameServer.Types.UserData;
+
+namespace Refresh.GameServer.Services;
+
+public class CommandService : EndpointService
+{
+    private readonly MatchService _match;
+    
+    public CommandService(LoggerContainer<BunkumContext> logger, MatchService match) : base(logger) {
+        this._match = match;
+    }
+
+    private readonly HashSet<ObjectId> _usersPublishing = new();
+
+    /// <summary>
+    /// Start tracking the user, eg. they started publishing
+    /// </summary>
+    /// <param name="id">The user ID</param>
+    public void StartPublishing(ObjectId id)
+    {
+        //Unconditionally add the user to the set
+        _ = this._usersPublishing.Add(id);
```

Comment:
This isn't pure so we dont need a blank assignment

Reply:
This was mostly done to be explicit about whats happening (ignoring the result) but the comment does that already, so i can remove the `_ =`